doc/source/security/encrypt-traffic.rst

```diff
@@ -281,7 +334,52 @@ Make sure that the following options are set in :file:`my.cnf` on all nodes:
    if keyring configuration across nodes is different.
 
 The following SST methods are available:
-``rsync``, ``mysqldump``, and ``xtrabackup``.
+``xtrabackup``, ``rsync``, and ``mysqldump``.
```

Comment:
This is a comment about the keyring_file information above.   It's a little misleading.

If the keyring is used, then SST encryption is required.
If SST encryption is used, there is no keyring requirement.

So the section should say something like, "if the keyring_file plugin is being used..."
It sounds like using the keyring is required for SST encryption, which is not true.